### ISSUE TITLE
1607558400

### ISSUE THREAD

**Issue by stringles** [MachineGunnur/GOG-Games#264]
The site has [the numbers](https://archive.is/0jFSi) instead of a text message for days. It looks like some epoch time. "2020-12-10," it said. What for, what will be? Maybe just some mistake happened.

**Comment by SilentMRG** [comment]
illuminati confirmed!

**Comment by MachineGunnur** [comment]
1607338800

:D

**Comment by Seeneres** [comment]
![Screenshot_2020-12-07-02-23-15](https://user-images.githubusercontent.com/75593144/101299781-ae148500-3833-11eb-99ca-aa218b3f40d0.png)

**Comment by SilentMRG** [comment]
Half Life 3 confirmed!

Cyberpunk will not be released until 2077!

These dates are just to raise expectations... Do not fall into the clutches of these conspirators.

**Comment by stringles** [comment]
Linking the previous date to "Cyberpunk 2077" probably makes sense, thanks. Now [said](https://archive.md/W0Rq4) the date and another  mentioned above, it means 11:16 of today, the current time. 15 minutes in, nothing occurs.

**Comment by MachineGunnur** [comment]
)) see you in 30 hours. 

btw if you think about supporting CDPR and buying the game. read this: https://www.gog.com/forum/general/preload_cyberpunk_2077_today_cfffe/post117

no-offline preload installers - forcing GOG Galaxy down throats of people- and locked single-player content behind a one-time online connection. 

remember [FCKDRM](https://www.fckdrm.com/)? really CDPR and GOG. comeon.

**Comment by 0xPaul** [comment]
DRM for me but not for thee.

**Comment by thepwrtank18** [comment]
Cyberpunk is here. Time to see what happens.

**Comment by optamiki** [comment]
There's gonna be an installer on gog-games.com soon or... what exactly?

**Comment by thepwrtank18** [comment]
Considering it's not on the [queue](https://gog-games.com/queue), probably not.

I think GOG is making it Galaxy-only for a few days, until the launch sales come in and they get to normal levels.
Then they will make an offline installer.

**Comment by DarkWorld25** [comment]
I mean like surely, even the steam version is drm free. Might be worth checking cs underground

**Comment by thepwrtank18** [comment]
All steam games have ([although really weak](https://gitlab.com/Mr_Goldberg/goldberg_emulator)) DRM on them.

**Comment by DarkWorld25** [comment]
nah apparently you can straight up launch the game without steam through the executable. On a side note, this is cs.rin.ru right now
https://i.imgur.com/ibgyMkD.png

**Comment by thepwrtank18** [comment]
Usually it will let you do that if Steam is running. If Steam launches itself (if it's not running in the background) when you open the executable, not DRM free. If so, DRM free!

**Comment by xathian** [comment]
> Usually it will let you do that if Steam is running. If Steam launches itself (if it's not running in the background) when you open the executable, not DRM free. If not, DRM free!

You were already wrong once, stop doubling down. DRM is not mandatory for a game on Steam, it's a choice and several games (Such as Witcher 3) don't have it, you can install the game then copy the files off to another PC or straight up uninstall steam and the game still launches from the exe.

There is even a thread on the GOG Forums cataloging all Steam games that are DRM free.

**Comment by Tene21** [comment]
> DRM is not mandatory for a game on Steam, it's a choice and several games (Such as Witcher 3) don't have it, you can install the game then copy the files off to another PC or straight up uninstall steam and the game still launches from the exe.

i thought this was common knowledge by now? this applies to EGS too, off the top of my head, my Steam copy of DMC5 is DRM-free, as are my EGS copies of Hades and Control.

**Comment by ghost-420** [comment]
> btw if you think about supporting CDPR and buying the game. read this: https://www.gog.com/forum/general/preload_cyberpunk_2077_today_cfffe/post117
Archives:
https://web.archive.org/web/20201210005230/https://www.gog.com/forum/general/preload_cyberpunk_2077_today_cfffe/post117
https://archive.is/EJECX

**Comment by Supernoodles1990** [comment]
> 
> 
> There's gonna be an installer on gog-games.com soon or... what exactly?



> 
> 
> Considering it's not on the [queue](https://gog-games.com/queue), probably not.
> 
> I think GOG is making it Galaxy-only for a few days, until the launch sales come in and they get to normal levels.
> Then they will make an offline installer.

https://www.gogalaxy.com/en/yourcyberpunk/ Confirms at the bottom of the page that offline installers go live release day so technically right now

**Comment by Tene21** [comment]
there are already torrents kicking about of the offline installer

**Comment by Supernoodles1990** [comment]
> 
> 
> there are already torrents kicking about of the offline installer

yep just found one guess that answers my question but 103gb without bonus content oof

**Comment by MachineGunnur** [comment]
was coming. was large and made sure it was intact and good for upload.

its coming now.

**Comment by MaxMega12** [comment]
ily MG

**Comment by Tene21** [comment]
at the risk of sounding impatient, is there any ETA?

**Comment by MachineGunnur** [comment]
After compress, maybe 15-20 minutes for zippyshare.

**Comment by MaxMega12** [comment]
website's being hammered; just bc users or other?

**Comment by DarkWorld25** [comment]
Do you have a total file size and file number count?

**Comment by MaxMega12** [comment]
> 
> 
> Do you have a total file size and file number count?

seems like 106.5 gigs, 213 files

**Comment by MachineGunnur** [comment]
raw gog files: 117,306,537,991 bytes, 44 files (game + goodies)

**Comment by Tene21** [comment]
seems like a couple of files were missed on the MegaUpload version - part109 and part111 are MIA

**Comment by MachineGunnur** [comment]
links are interchangeable. use zippy for missing or others once up. megaup is slow sometimes. only 3 chances to upload each file then moves onto next. happens.

**Comment by Tene21** [comment]
zippy is blocked on the ISP-level in my country, sadly. i can probably TOR around to grab it

**Comment by MachineGunnur** [comment]
free vpn then, protonvpn.

**Comment by DarkWorld25** [comment]
Change your DNS. Most ISP level block relies on default DNS

**Comment by Tene21** [comment]
> Change your DNS. Most ISP level block relies on default DNS

already done, still get a 403 on every zippy page unless i'm using a VPN

**Comment by stringles** [comment]
Zippyshare itself blocks.

**Comment by Ojskin** [comment]
noooo :( missing offline files so close yet so far, how can i fix

**Comment by Ic3man101** [comment]
is the 1ficher link working?

**Comment by d0x7** [comment]
@Ic3man101 Yes, it is. It's one part tho and not 200+, therefore not interchangeable with the other ones.. I noticed that after I downloaded 40 parts (I was waiting for 1ficher and thought it'd be parts on 1ficher too).
![image](https://user-images.githubusercontent.com/17951467/101739068-5130fd00-3ac7-11eb-8fac-bb9470114fe0.png)

**Comment by DarkWorld25** [comment]
I worked out which files are down on zippy:
part 2
part 25
part 37
part 62
part 79
part 184
part 186
part 199

**Comment by DarkWorld25** [comment]
72 also down

**Comment by ajhix36** [comment]
Downloaded from 1filchier. Integrity check failed in official installer. setup_cyberpunk_2077_build_3214677change_4155897_(64bit)_(43299)-6.bin

**Comment by MaxMega12** [comment]
same here w/ 1filchier, csrin reports similar issues

**Comment by thepwrtank18** [comment]
jesus, that's even worse than Prey (80+ parts)

**Comment by stringles** [comment]
Read [the issue here](https://github.com/MachineGunnur/GOG-Games/issues/268), they have supplied a fix. But looking at the buggy/glitchy state of the game, I expect many, many patches and reuploads. Myself wouldn't even run it unil the devs made it nice and far more stable.

> "According to stats available from Steam, at one point this morning 1,003,264 players were roaming Night City. For comparison, the massively popular Counter-Strike: Global Offensive has an all-time peak of 1.3m, and that’s been around for eight years," [Torrentfreak](https://torrentfreak.com/pirating-cyberpunk-2077-cd-projekt-sends-waves-of-dmca-notices-201210/)

This is insane. No wonder it made issues to, the number is quite high.

**Comment by 0xPaul** [comment]
The 1.05 patch is not compatible with the posted 1.04 patch (mismatched build numbers).

**Comment by d0x7** [comment]
> The 1.05 patch is not compatible with the posted 1.04 patch (mismatched build numbers).
> 
> ```
> 1.04: patch_cyberpunk_2077_Build_3214677Change_4155897_v1_(43299)_to_Build_3217599Change_4158334_(43399).exe
> 1.05: patch_cyberpunk_2077_1.04_(43336)_to_Build_3233390Change_4173226_(43550).exe
> ```
> 
> Notice that 43399 is newer than 43336.
> 


While I can confirm that the order doesn't make sense, and the patch should be for 43399 (and not 43336?), it worked for me. My game was on the 1.04 patch and updated without a problem. Wish I could help you with your problem. Are you on the pre-latest patch or on the original game?

![image](https://xiam.li/i/rmH7P) 
![image](https://xiam.li/i/PiQXM)

**Comment by 0xPaul** [comment]
I applied 1.04 twice, since one file failed the first time. Applying it again just caused even more failures.

**Comment by Masamune3210** [comment]
Are you making sure to get all of the installers from here and not somewhere else? Downloaded the game and both patches and it worked just fine after chewing my hard drive for about a hour per patch
